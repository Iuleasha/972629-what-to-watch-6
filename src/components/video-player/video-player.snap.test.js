import {render} from '@testing-library/react';
import {createMemoryHistory} from 'history';
import React from 'react';
import {Router} from 'react-router-dom';
import VideoPlayer from './video-player';


describe(`Test VideoPlayer`, () => {
  const history = createMemoryHistory();

  test(`Should video preview render correctly`, () => {
    const {container} = render(
        <Router ver history={history}>
          <VideoPlayer hasCustomControls={false} isAutoPlay={true} isMuted={true} src={``} filmId={`1`}/>
        </Router>,
    );

    expect(container).toMatchSnapshot();
  });

  test(`Should video page render correctly with custom controls`, () => {

    const {container} = render(
        <Router history={history}>
          <VideoPlayer hasCustomControls={false} isAutoPlay={true} isMuted={true} src={``} filmId={`1`}/>
        </Router>,
    );

    expect(container).toMatchSnapshot();
  });
});
