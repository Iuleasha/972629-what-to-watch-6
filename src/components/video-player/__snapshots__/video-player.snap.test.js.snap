// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test VideoPlayer Should video page render correctly with custom controls 1`] = `
<div>
  <video
    autoplay=""
    class="player__video"
    src=""
  />
</div>
`;

exports[`Test VideoPlayer Should video preview render correctly 1`] = `
<div>
  <video
    autoplay=""
    class="player__video"
    src=""
  />
</div>
`;
